### ISSUE TITLE
Incorrect Download Link for T2_IDS_199_217.tar.gz

### ISSUE THREAD

**Issue by 5o1** [cai2r/fastMRI_prostate#20]
2025/01/09 update:

The file has now been downloaded successfully and all files have passed sha256 validation

Three of the files in `breast` and most of the files in `prostate` failed to extract successfully. It looks like compressed packets are corrupted at distribution source (sha256 check passed)

error message:
```bash
gzip: stdin: unexpected end of file                                              
tar: Unexpected EOF in archive                                                   
tar: Unexpected EOF in archive                                                   
tar: Error is not recoverable: exiting now
```

Most files from `prostate` cannot be extracted

failed files in breast:
```bash
Failed to extract file ./fastMRI_breast_IDS_231_240.tar.gz!
Failed to extract file ./fastMRI_breast_IDS_271_280.tar.gz!
Failed to extract file ./fastMRI_breast_IDS_061_070.tar.gz!
```

=====
There are two wrong download links provided for the file `fastMRI_prostate_T2_IDS_199_217%20.tar.gz` in the mail "NYU FastMRI Data Download Links". When I download this file, I get the response "The specified key does not exist".

This error appears in two places:

- In the hyperlink in the first section.
- After the "`curl -C ...`" in the second section.

In the Breast dataset, the `fastMRI_breast_labels` file also has the same error.

I also sent an email to fastmri@med.nyu.edu, but I'm not sure if this mailbox is looked after.

**Comment by ruveydayilmaz0** [comment]
> 2025/01/09 update:
> 
> The file has now been downloaded successfully and all files have passed sha256 validation
> 
> Three of the files in `breast` and most of the files in `prostate` failed to extract successfully. It looks like compressed packets are corrupted at distribution source (sha256 check passed)
> 
> error message:
> 
> gzip: stdin: unexpected end of file                                              
> tar: Unexpected EOF in archive                                                   
> tar: Unexpected EOF in archive                                                   
> tar: Error is not recoverable: exiting now
> Most files from `prostate` cannot be extracted
> 
> failed files in breast:
> 
> Failed to extract file ./fastMRI_breast_IDS_231_240.tar.gz!
> Failed to extract file ./fastMRI_breast_IDS_271_280.tar.gz!
> Failed to extract file ./fastMRI_breast_IDS_061_070.tar.gz!
> ===== There are two wrong download links provided for the file `fastMRI_prostate_T2_IDS_199_217%20.tar.gz` in the mail "NYU FastMRI Data Download Links". When I download this file, I get the response "The specified key does not exist".
> 
> This error appears in two places:
> 
> * In the hyperlink in the first section.
> * After the "`curl -C ...`" in the second section.
> 
> In the Breast dataset, the `fastMRI_breast_labels` file also has the same error.
> 
> I also sent an email to [fastmri@med.nyu.edu](mailto:fastmri@med.nyu.edu), but I'm not sure if this mailbox is looked after.

Hi @5o1 , have you found a solution to this error?

**Comment by 5o1** [comment]
> > 2025/01/09 update:
> > 
> > The file has now been downloaded successfully and all files have passed sha256 validation
> > 
> > Three of the files in `breast` and most of the files in `prostate` failed to extract successfully. It looks like compressed packets are corrupted at distribution source (sha256 check passed)
> > 
> > error message:
> > 
> > gzip: stdin: unexpected end of file                                              
> > tar: Unexpected EOF in archive                                                   
> > tar: Unexpected EOF in archive                                                   
> > tar: Error is not recoverable: exiting now
> > Most files from `prostate` cannot be extracted
> > 
> > failed files in breast:
> > 
> > Failed to extract file ./fastMRI_breast_IDS_231_240.tar.gz!
> > Failed to extract file ./fastMRI_breast_IDS_271_280.tar.gz!
> > Failed to extract file ./fastMRI_breast_IDS_061_070.tar.gz!
> > ===== There are two wrong download links provided for the file `fastMRI_prostate_T2_IDS_199_217%20.tar.gz` in the mail "NYU FastMRI Data Download Links". When I download this file, I get the response "The specified key does not exist".
> > 
> > This error appears in two places:
> > 
> > * In the hyperlink in the first section.
> > * After the "`curl -C ...`" in the second section.
> > 
> > In the Breast dataset, the `fastMRI_breast_labels` file also has the same error.
> > 
> > I also sent an email to [fastmri@med.nyu.edu](mailto:fastmri@med.nyu.edu), but I'm not sure if this mailbox is looked after.
> 
> Hi @5o1 , have you found a solution to this error?

No. I'm not sure if they have solved this problem since I submitted this issue. Now I don't do MRI research anymore. I'm sorry I didn't help you.
